### PR TITLE
[usecase] Automate 4 usecases

### PR DESCRIPTION
**Pull request by JianfengXu**
Impacted TCs num(approved): New 0, Update 4, Delete 0
Unit test Platform: Andriod
Unit test result summary: Pass 4, Fail 0, Blocked 0

### CODE REVIEW

**Comment by Honry** [comment]
You must go into frame first.  It is already implemented in steps.py.
BTW, the reason you provide is neither a WebDriver issue nor a test case issue, we could not accept it.

**Comment by JianfengXu** [comment]
I have updated webmessaging.feature, and the usecase can be pass now.

File: tools/atip/atip/web/web.py [hunk 1]
```diff
@@ -209,6 +209,23 @@ def __check_normal_text(self, text, display=True):
             print "Failed to get element: %s" % e
         return None
```
Comment (Honry) [inline tools/atip/atip/web/web.py]: web.py is a public library for all BDD tests, every change may affect other tests, so we should be very careful on web.py update. 
Please put the new add/update function into $suite/../steps.py or you can create a new web.py under the test suite.
And finally merge them after a comprehensive review and improvement.

File: usecase/usecase-webapi-xwalk-tests/tests.full.xml [hunk 2]
```diff
@@ -13,22 +13,52 @@
...
+        <description>
+          <test_script_entry test_script_expected_result="0">/opt/usecase-webapi-xwalk-tests/bddrunner testscripts/IndexedDB.feature</test_script_entry>
+        </description>
+      </testcase>
       <testcase component="Crosswalk Use Cases/WebAPI" execution_type="auto" id="RawSockets" platform="all" priority="P0" purpose="Raw Sockets" status="approved" type="functional_positive">
         <description>
           <test_script_entry test_script_expected_result="0">/opt/usecase-webapi-xwalk-tests/bddrunner testscripts/RawSockets.feature</test_script_entry>
         </description>
       </testcase>
+      <testcase component="sample" type="functional_positive" status="approved" execution_type="manual" platform="all" priority="P0" id="SessionHistory" purpose="Session History">
```
Comment (Honry) [inline usecase/usecase-webapi-xwalk-tests/tests.full.xml]: The execution_type should be 'auto'

File: usecase/usecase-webapi-xwalk-tests/testscripts/steps/steps.py [hunk 3]
```diff
@@ -12,6 +12,11 @@ def i_visit_frame(context, key):
     context.app.driver.switch_to_frame(key)
     assert True
 
+@step(u'I jump frame')
```
Comment (Honry) [inline usecase/usecase-webapi-xwalk-tests/testscripts/steps/steps.py]: This step is not only specific for frame, can we rename it a common name?

**Comment by Honry** [comment]
Please read all the comments carefully, you just update one place. 
For tools/atip/atip/web/web.py, this solution cannot meet the requirement, let's find another way.

File: usecase/usecase-webapi-xwalk-tests/tests.android.xml [hunk 4]
```diff
@@ -11,18 +11,39 @@
     <set name="Networking &amp;amp; Storage" background="brand-warning" icon="glyphicon-link" type="script">
       <testcase component="Crosswalk Use Cases/WebAPI" execution_type="manual" id="FileSystem" purpose="File System">
       </testcase>
+      <testcase component="sample" execution_type="auto" id="IndexedDB" purpose="IndexedDB">
```
Comment (zqzhang) [inline usecase/usecase-webapi-xwalk-tests/tests.android.xml]: Any story of the `component="sample"`?

Comment (JianfengXu) [inline usecase/usecase-webapi-xwalk-tests/tests.android.xml]: component="sample" come from demo-express, mean this sample come from demo-express,  is not usecase.

File: usecase/usecase-webapi-xwalk-tests/testscripts/SessionHistory.feature [hunk 5]
```diff
@@ -0,0 +1,17 @@
+Feature: Usecase WebAPI
+ Scenario: Networking &; Storage/SessionHistory Test
```
Comment (Honry) [inline usecase/usecase-webapi-xwalk-tests/testscripts/SessionHistory.feature]: Please remove the ';'

File: usecase/usecase-webapi-xwalk-tests/tests.full.xml [hunk 6]
```diff
@@ -13,22 +13,42 @@
...
+      <testcase component="sample" type="functional_positive" status="approved" execution_type="manual" platform="all" priority="P0" id="WebStorage" purpose="Web Storage">
+        <description>
+          <test_script_entry test_script_expected_result="0">/opt/usecase-webapi-xwalk-tests/bddrunner testscripts/WebStorage.feature</test_script_entry>
+        </description>
       </testcase>
       <testcase component="Crosswalk Use Cases/WebAPI" execution_type="manual" id="WebSpeech" platform="tizen" priority="P0" purpose="WebSpeech" status="approved" type="functional_positive">
       </testcase>
     </set>
     <set name="Performance &amp;amp; Optimization" background="brand-danger" icon="glyphicon-th-large" type="script">
+      <testcase component="sample" type="functional_positive" status="approved" execution_type="manual" platform="all" priority="P0" id="NavigationTiming" purpose="Navigation Timing">
```
Comment (Honry) [inline usecase/usecase-webapi-xwalk-tests/tests.full.xml]: This usecase should be auto

File: usecase/usecase-webapi-xwalk-tests/tests.full.xml [hunk 7]
```diff
@@ -13,22 +13,42 @@
...
+      </testcase>
+      <testcase component="Crosswalk Use Cases/WebAPI" execution_type="manual" id="SpeechSynthesis" platform="tizen" priority="P0" purpose="SpeechSynthesis" status="approved" type="functional_positive">
+      </testcase>
       <testcase component="Crosswalk Use Cases/WebAPI" execution_type="auto" id="WebSocket" platform="all" priority="P0" purpose="WebSocket" status="approved" type="functional_positive">
         <description>
           <test_script_entry test_script_expected_result="0">/opt/usecase-webapi-xwalk-tests/bddrunner testscripts/WebSocket.feature</test_script_entry>
         </description>
       </testcase>
-      <testcase component="Crosswalk Use Cases/WebAPI" execution_type="manual" id="SpeechSynthesis" platform="tizen" priority="P0" purpose="SpeechSynthesis" status="approved" type="functional_positive">
+      <testcase component="sample" type="functional_positive" status="approved" execution_type="manual" platform="all" priority="P0" id="WebStorage" purpose="Web Storage">
```
Comment (Honry) [inline usecase/usecase-webapi-xwalk-tests/tests.full.xml]: This should be auto.

File: usecase/usecase-webapi-xwalk-tests/samples/WebCL/css/style.css [hunk 8]
```diff
@@ -0,0 +1,40 @@
+/*
```
Comment (Honry) [inline usecase/usecase-webapi-xwalk-tests/samples/WebCL/css/style.css]: Why you add this file in Automation PR?

**Comment by Honry** [comment]
LGTM, @zqzhang @jennycao , any concern?

**Pull request merged**